draft-packages/tooltip/KaizenDraft/Tooltip/README.mdx

```diff
@@ -33,104 +35,116 @@ import WhenNotToUse from "docs-components/WhenNotToUse"
 
 ## To keep in mind
 
-Use sparingly. Tooltips innately hide information. Consider exposing the information immediately without a tooltip or removing it completely.
+* Use sparingly. Tooltips innately hide information. Consider exposing the information immediately without a tooltip or removing it completely.
+* Tooltips have slightly different guidelines to Popovers and Modals. For more information, see [Tooltip, Popover and Modal guidelines.](https://cultureamp.design/guidelines/tooltip-popover-modal/)
+* If the information is crucial for a user to proceed, consider using a [modal](https://cultureamp.design/components/modal) or [inline notification](https://cultureamp.design/guidelines/notifications) instead.
 
 ### Anatomy
 
-*   The anatomy of a tooltip is made up of a **container**, **description** and **tip** (i.e. arrow).
-*   A tooltip is connected to a **trigger** element, such as a button or icon, that shows the tooltip when the interactive trigger element is hovered, focused, or clicked. (More details below.)
+* The anatomy of a tooltip is made up of a container, description and arrow.
+    * They do not include a title or heading. If you need to include a title, use a [Popover](https://cultureamp.design/components/popover/)
+* A tooltip is connected to a trigger element, such as a button or icon, that shows the tooltip when the interactive trigger element is hovered or focused. (More details below.)
 
-### Visual details
+### Moods
 
-*   Text and tips within tooltips can be left, centered and right aligned, as appropriate within its environmental context.
-*   Use strong signifiers for trigger elements to ensure the tooltip is discoverable.
-*   The proximity of tooltips is always paired nearby the element with which they are associated.
-*   Don’t crop tooltips e.g. at the edge of modals.
+For context, see [Moods](https://cultureamp.design/guidelines/moods).
 
-### Trigger
+* When combined with an icon, use the matching icon and mood (Positive, Informative, Cautionary and Highlight). For example, when using a Blue information icon, show an informative tooltip.
+* When applied without a mood-based trigger, use the Default tooltip (white).
```

Comment:
Now that I see upper case "Default" here, do those refer to very specific states so that capitalization makes sense?

Reply:
That's right, yep.  The use of capitalization is because **Default** here is in reference to the mood.